app/src/main/java/org/fossasia/openevent/general/search/SearchLocationFragment.kt

```diff
@@ -34,6 +36,9 @@ class SearchLocationFragment : Fragment() {
     override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View? {
         rootView = inflater.inflate(R.layout.fragment_search_location, container, false)
 
+        val imm = context?.getSystemService(Context.INPUT_METHOD_SERVICE) as? InputMethodManager
+        imm?.showSoftInput(view, 0)
```

Comment:
Same method is written in utils. You can call it from there. :+1:

Reply:
@liveHarshit I tried doing it that way, but then the keyboard did not show up at all.

Reply:
Okay, let me check. 
What is view here, pass it. And change `InputMethodManager.SHOW_FORCED` to `InputMethodManager.RESULT_UNCHANGED_SHOWN` in utils which contains constant value 0. Then it should work.

Reply:
@liveHarshit I replaced lines 39 and 40 in searchLocationFragment with `Utils.showSoftKeyboard(context, rootView)`. In Utils also I made the change that you mentioned above. But the keyboard still does not pop up.

Reply:
`imm?.showSoftInput(view, 0)`
What is view here?

Reply:
That is View?, not view. It is from getView().

Reply:
Ok then change view to nullable in utils, same as context.

Reply:
It works. I have not made any changes in the onResume() function. It is still `Utils.showSoftKeyboard(context, rootview)`

Reply:
But remove it! It was written for the same purpose earlier.

Reply:
I removed it, it works fine. But now when the user presses on the back button on the top of the screen, the keyboard does not go down. It stays up in the main activity and back button on the nav bar has to pressed to close the keyboard.

Reply:
@liveHarshit Should I revert back to the initial commit I made? That way the keyboard was going down automatically even when the user went to the main activity by pressing the back button in the top.

Reply:
> And change `InputMethodManager.SHOW_FORCED` to `InputMethodManager.RESULT_UNCHANGED_SHOWN` in utils which contains constant value 0. Then it should work.

You forgot these change. Do it and your initial changes and these changes become same,